velox/common/base/tests/ScratchTest.cpp

```diff
@@ -35,7 +35,6 @@ TEST(ScratchTest, basic) {
   {
     ScratchPtr<int32_t> ints(scratch);
     ScratchPtr<int64_t> longs(scratch);
-    auto tempLongs = longs.get(2000);
```

Comment:
You have to put this back in, even though the variable is not used but `longs.get(2000)` is allocating memory which then used to verify allocation behavior below. Removing this is causing a failure in this test.

Reply:
Seems like we should have a comment here explaining this (is there a way to add an annotation to prevent the compiler from thinking the variables is unused?)